image_data.hh

```diff
@@ -1,3 +1,5 @@
+#pragma once  // REFACTOR: Replaced #ifndef VT_IMAGE_DATA_HH guard with modern pragma once
+
```

Comment:
please move down to where the `#ifndef` include guard was and remove REFACTOR comment